Error: 'AF_INET' macro redefined [-Werror,-Wmacro-redefined]

When I try to compile the **packet-solution** directory using: **sudo make** it throws this error:

> xdp_prog_kern_03.c:214:9: error: 'AF_INET' macro redefined [-Werror,-Wmacro-redefined]
> #define AF_INET 2
>         ^
> /usr/include/bits/socket.h:96:9: note: previous definition is here
> 
> #define AF_INET         PF_INET
>         ^
> 1 error generated.
> make: *** [../common/common.mk:111: xdp_prog_kern_03.o] Error 1

It also happens with: 

> xdp_prog_kern_03.c:215:9: error: 'AF_INET6' macro redefined [-Werror,-Wmacro-redefined]
> #define AF_INET6 10
>         ^
> /usr/include/bits/socket.h:104:9: note: previous definition is here
> 
> #define AF_INET6        PF_INET6
>         ^
> 1 error generated.
> make: *** [../common/common.mk:111: xdp_prog_kern_03.o] Error 1

I simply commented those lines but i don't know if this is normal to happen.

Thanks!

--- a/packet-solutions/xdp_prog_kern_03.c
+++ b/packet-solutions/xdp_prog_kern_03.c
@@ -211,7 +211,9 @@ out:
 	return xdp_stats_record_action(ctx, action);
 }
 
+#undef AF_INET
 #define AF_INET 2
+#undef AF_INET6
 #define AF_INET6 10
 #define IPV6_FLOWINFO_MASK bpf_htonl(0x0FFFFFFF)
 

